mettle/src/stdapi/net/resolve.c

```diff
@@ -21,8 +21,10 @@ void resolve_host_async(struct eio_req *req)
 	int ret_val = TLV_RESULT_SUCCESS;
 
 	uint32_t addr_type;
-	if (tlv_packet_get_u32(ctx->req, TLV_TYPE_ADDR_TYPE, &addr_type) ||
-			(addr_type != AF_INET && addr_type != AF_INET6)) {
+	tlv_packet_get_u32(ctx->req, TLV_TYPE_ADDR_TYPE, &addr_type);
+	if (addr_type == 23) {
```

Comment:
Can we copy a similar pattern to the python/php implementations here? 👀 

i.e. a named constant mapping both win ipv4 and win ipv6 to the native af_inet/af_inet6

Reply:
Got that updated, spent some time looking for a place to add these constants outside of this file so they could be used across the project. However didn't find a _nice_ home for them, so any thoughts on where they'd be best suited?